treeherder/log_parser/tasks.py

```diff
@@ -1,11 +1,16 @@
+import json
```

Comment:
This import should be corrected.
It's only used in the `extract_text_log_artifacts()` function, from the old log_parser/utils.py module. But back there, the function was actually using `simplejson`.

Reply:
It is a reminiscence from moving the functions which caused circular imports.